A hack for 2.4 HPPA.  I don't completely understand what this does or why
it helps (and whether it's still needed).

--- openafs-1.3.87.orig/src/lwp/process.c
+++ openafs-1.3.87/src/lwp/process.c
@@ -101,7 +101,9 @@
 #elif	defined(AFS_HPUX_ENV)
 #define	LWP_SP	1
 #elif	defined(AFS_LINUX20_ENV)
-#if defined(AFS_PPC_LINUX20_ENV) || defined(AFS_PPC64_LINUX20_ENV)
+#if defined(AFS_PARISC_LINUX24_ENV )
+#define	LWP_SP	1
+#elif defined(AFS_PPC_LINUX20_ENV)|| defined(AFS_PPC64_LINUX20_ENV)
 #define LWP_SP 0
 #elif   defined(AFS_I386_LINUX20_ENV)
 #define LWP_SP 4
